CHANGELOG.md

```diff
@@ -2,7 +2,9 @@
 
 # Changelog
 
-<!-- next version -->
+### 💡 Enhancements 💡
+
+- `k8sattributesprocessor`: Default `container.id` to latest container instance unless `k8s.container.restart_count` is specified. (#16432)
```

Comment:
Sorry for the delay. Please add changelog entry to https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/.chloggen 

This file is auto-generated